### PR TITLE
Multitenant

### PR DESCRIPTION
**Pull request by jreynard-code**
## What's new
- Adapt cosmotech-api helm chart for multi-tenancy
- Adapt local deployment script for multi-tenancy

### CODE REVIEW

**Review by mahkoCosmo — APPROVED** [review]
LG2M

File: .github/scripts/test-helm-release.sh [hunk 1]
```diff
@@ -20,15 +20,15 @@ kubectl describe all --all-namespaces
...
+  kubectl -n "${CHART_RELEASE_TEST_NAMESPACE}" logs "cosmotech-api-${CHART_RELEASE_TEST_NAMESPACE}-${API_VERSION}-test-connection-${test}"
   echo "-"
 done
 
 if [[ "${retVal}" != "0" ]]; then
   echo "Helm Release testing did not complete successfully: $retVal."
-  echo "  Command: helm -n ${CHART_RELEASE_TEST_NAMESPACE} test cosmotech-api-${API_VERSION}"
-  exit $retVal
+  echo "  Command: helm -n ${CHART_RELEASE_TEST_NAMESPACE} test cosmotech-api-${CHART_RELEASE_TEST_NAMESPACE}-${API_VERSION}"
+  exit $retVald
```
Comment (lalepee) [inline .github/scripts/test-helm-release.sh:31]: are you sure about that ? retVald ? not retVal

**Review by lalepee — COMMENTED** [review]
Commit messages could have been reworked to be more clear

**Review by lalepee — CHANGES_REQUESTED** [review]
retVald must be changed

**Review by lalepee — APPROVED** [review]

**Pull request merged**